libs/vre/resource-editor/resource-properties/src/lib/incoming-links-property.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { ReadResource, ReadResourceSequence } from '@dasch-swiss/dsp-js';
+import { AppError } from '@dasch-swiss/vre/core/error-handler';
+import { DspResource } from '@dasch-swiss/vre/shared/app-common';
+import { IncomingService } from '@dasch-swiss/vre/shared/app-common-to-move';
+import { Observable, of } from 'rxjs';
+import { expand, map, reduce, take, takeWhile } from 'rxjs/operators';
+import { IncomingOrStandoffLink } from './incoming-link.interface';
+import { sortByKeys } from './sortByKeys';
+
+@Component({
+  selector: 'app-incoming-links-property',
+  template: `
+    <app-property-row
+      [tooltip]="'resource.incomingLink.tooltip' | translate"
+      [label]="'resource.incomingLink.label' | translate"
+      [borderBottom]="true"
+      [isEmptyRow]="!loading && allIncomingLinks.length === 0">
+      <ng-container *ngIf="allIncomingLinks.length > 0; else loadingTemplate">
```

Comment:
remove "else loadingTemplate" as the progress indicator spins eternally if there is no incoming links to a resource